samples/client/petstore-security-test/java/okhttp-gson/build.gradle

```diff
@@ -94,10 +94,11 @@ if(hasProperty('target') && target == 'android') {
 }
 
 dependencies {
-    compile 'io.swagger:swagger-annotations:1.5.15'
-    compile 'com.squareup.okhttp:okhttp:2.7.5'
-    compile 'com.squareup.okhttp:logging-interceptor:2.7.5'
+    compile 'io.swagger:swagger-annotations:1.5.17'
+    compile 'com.squareup.okhttp3:okhttp:3.14.2'
+    compile 'com.squareup.okhttp3:logging-interceptor:3.14.2'
     compile 'com.google.code.gson:gson:2.8.1'
+    compile 'io.gsonfire:gson-fire:1.8.0'
```

Comment:
There are some additional changes in `petstore-security-test` that don't seem related to the OkHttp 3.x change. I'm guessing something wasn't updated in a previous commit.